circe/pricing_push/evaluate.py

```diff
@@ -75,10 +77,14 @@ def evaluate_sequential():
                 break
 
 
-    print('---- Finish generating sequential input files')
+    logging.debug('---- Finish generating sequential input files')
     
 
 if __name__ == '__main__':
+    global logging
```

Comment:
Is this file always run directly `python3 evaluate.py`?

Reply:
This file runs after the whole system is deployed. It is ran directly by the start script.